repo/el9-x86_64-sap-rhui-azure.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Chm, what's this change?

Reply:
Excellent question.  I don't know.  I ran the make and checked everything in without looking.  Let me see if it's a local leftover of some sort.

Reply:
Nope.  Wasn't me.  It's configured in `repo-definitions.yaml` but missing from `main`.

Reply:
v

Reply:
Split the commits.

Reply:
Hehe, I'm sorry.